realsense2_camera/scripts/rs2_test.py

```diff
@@ -300,7 +300,7 @@ def run_tests(tests):
             cmd_params = ['roslaunch', 'realsense2_camera', 'rs_from_file.launch'] + params_str.split(' ')
             print ('running command: ' + ' '.join(cmd_params))
             p_wrapper = subprocess.Popen(cmd_params, stdout=None, stderr=None)
-            time.sleep(2)
+            time.sleep(4)

```

Comment:
We're not entirely sure if this change is a good idea, but locally this change fixed the test failure.